packages/core/src/types/next-fields.ts

```diff
@@ -14,6 +14,9 @@ export type ListGraphQLTypes = { types: GraphQLTypesForList };
 export type FieldData = {
   lists: Record<string, ListGraphQLTypes>;
   provider: DatabaseProvider;
+  assets: {
+    getStorage: (storage: string) => StorageKind | undefined;
```

Comment:
This returns the storage config not the kind, right?

Reply:
This `getStorage` function returns a storageKind, and is used for looking up the full info of storage.

Previously here we had a `getStorageKind` function that would get `s3` or `local` but since we occasionally needed more than that we just get the whole object.

Reply:
Oh, sorry, I expected `StorageKind` to be `'s3' | 'local'` based on the name. Makes sense

Reply:
@mitchellhamilton the naming of the types for this probably needs a bike-shed moment